packages/coding-components/templates/webpack/webpack.config.js

```diff
@@ -14,8 +14,6 @@
  */
 
 const HtmlWebPackPlugin = require("html-webpack-plugin");
```

Comment:
While we're removing plugins, it may be time to remove this plugin as well. WDYT?

Reply:
Good point. I need to look into doing the same thing for the jsapi-resources webpack sample.

Reply:
update: decision was made to keep the html plugin around.